src/components/__snapshots__/Avatar.test.tsx.snap

```diff
@@ -36,6 +36,7 @@ exports[`Avatar renders correctly with address and name but no number 1`] = `
             "height": 40,
             "width": 40,
           },
+          undefined,
```

Comment:
The `undefined` snapshot updates are because the borderColor style on the ContactCirle component